### DISCUSSION TITLE
Can the 'Active Job' of a runner be made available in the Github API?

### DISCUSSION THREAD

**Discussion by JackCoplandGranta** [community/community#48977] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

So far as I'm aware the only way to see what job is active on a runner is to manually go to the specific runner page in settings:
![image](https://user-images.githubusercontent.com/43538264/222415253-b1e86cdf-7663-4c46-9bfa-cedb5f02639e.png)
Can this information be made accessible via the API? We are managing a lot of self hosted runner and this would be useful to understand the job load on those machines. Please correct me if I've just missed an endpoint somewhere!

**Comment by divyaGrvgithub** [comment]
Can this information be made accessible via the API? We are managing a lot of self hosted runner and this would be useful to understand the job load on those machines. Please correct me if I've just missed an endpoint somewhere!

**Comment by yakirm-cr** [comment]
It would be very useful if the information listed above will be available via this REST APIs:
1. https://docs.github.com/en/enterprise-cloud@latest/rest/actions/self-hosted-runners?apiVersion=2022-11-28#get-a-self-hosted-runner-for-an-enterprise
2. https://docs.github.com/en/enterprise-cloud@latest/rest/actions/self-hosted-runners?apiVersion=2022-11-28#get-a-self-hosted-runner-for-an-organization
3. https://docs.github.com/en/enterprise-cloud@latest/rest/actions/self-hosted-runners?apiVersion=2022-11-28#get-a-self-hosted-runner-for-a-repository

Maybe add the following fields to the output:
1. `active_job_url` - with a URL leading to the active job REST API URL or empty when there is no active job.
2. `active_job_labels` - list of labels for the active job (empty when there is no active job).

The "List self-hosted runners for..." REST APIs should also be considered for the above change.
What do you think?
